lib/composites/modals/style.less

```diff
@@ -28,18 +28,28 @@
       border-top-left-radius: 3px;
       border-top-right-radius: 3px;
 
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6,
+      @{prefix}close {
```

Comment:
didn't we talk about a helper class for this?  you mind adding `@{prefix}header-item`